service/policy/db/namespaces.go

```diff
@@ -160,11 +215,16 @@ func (c PolicyDBClient) UnsafeUpdateNamespace(ctx context.Context, id string, na
 	nsFqn := c.upsertAttrFqn(ctx, attrFqnUpsertOptions{namespaceID: id})
 	c.logger.Debug("upserted fqn for unsafely updated namespace", slog.Any("fqn", nsFqn))
 
-	attrs, err := c.ListAttributes(ctx, StateAny, id)
+	// TODO: deprecate the list of attributes and move upsert to a transaction/trigger
```

Comment:
This todo is resolved by the combination of these two PRs (and they should be merged first with any conflicts resolved in this PR):
- https://github.com/opentdf/platform/pull/1678
- https://github.com/opentdf/platform/pull/1679